scripts/contentScript.js

```diff
@@ -212,6 +221,46 @@ async function showFloatingActionButton(orgName, orgRepo) {
 	}
 }
 
+// showing the important files in a pr
+async function showImpFileInPr(repoOwner,repoName,userId) {
+	const body = {
+		"repo_owner": repoOwner,
+		"repo_name": repoName,
+		"user_id": userId,
+		"is_github": true
```

Comment:
Does this call not require the id of the pull request?

Reply:
Guessing this is resolved